addons/bus/static/src/services/bus_service.js

```diff
@@ -64,6 +64,7 @@ export const busService = {
             const { type } = messageEv.data;
             let { data } = messageEv.data;
             if (type === "notification") {
+                data.forEach((d) => (d.message.id = d.id)); // put notification id in notif payload
```

Comment:
Misleading comment, it's not in "payload", as payload is a specific key of message. (and it would be incorrect to put it in payload as it could conflict with actual data)